lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -99,10 +100,19 @@ def write_playbook
 
       def start_ansible_runner
         command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        command << verbosity_option if verbose?
         initialize_command(*command)
         logger.debug("[foreman_ansible] - Running command '#{command.join(' ')}'")
       end
 
+      def verbosity_option
+        '-' + 'v' * @verbosity_level.to_i
```

Comment:
what happens if there is `@verbosity_level > 5`? can ansible deal with `-vvvvv`? won't it fail? Wouldn't it be better to limit it?
```suggestion
        '-' + 'v' * [@verbosity_level.to_i,4].min
```

Reply:
@ezr-ondrej verbosity_level is the value derived from Ansible Settings https://github.com/theforeman/foreman_ansible/blob/master/app/models/setting/ansible.rb#L39-L52 which can't be more than 4. This value is exposed during the Ansible run https://github.com/theforeman/foreman_ansible/blob/1e040a065a97daa84d1a70cb90c396ec02f43e80/app/models/foreman_ansible/ansible_provider.rb#L26 So, its safe to assume that `verbosity_level` would always in the range of 0-4 (where 0 means disabled).

Reply:
:+1: I'm ok with that, just it's always good to be on the safe side, in case we allow some other places for the same input, but no biggie :)